man/man1/p11sak.1.in

```diff
@@ -107,6 +109,9 @@ attribute of the key and
 .B \-\-attr
 .IR [P M R L S E D G V W U A X N T]
 can be used to set the binary attributes of the key (see below for detailed description of the attributes).
+.B \-\-custom\-attr | \-c 
+.IR [[pub_attr]:[prv_attr]]
+can be used to set the binary attributes of the key (see below for deteiled describtion ans syntax).
```

Comment:
deteiled describtion -> detailed description

Reply:
ans syntax -> and syntax

Reply:
... more places below ...

Reply:
`binary attributes` -> `boolean attributes` please

Reply:
changed to CK_BBOOL